frontend/src/store/atoms.ts

```diff
@@ -0,0 +1,20 @@
+// Basic state should be here
+import { atom, RecoilState, selector } from "recoil";
+import { Methodology, Project } from "./types";
+import Api from "../api/api";
+
+export const projectsAtom: RecoilState<Project[]> = atom({
+  key: "Projects",
+  default: selector({
+    key: "Projects/Default",
+    get: () => Api.fetchProjects(),
+  }),
```

Comment:
lol, it will do exactly opposite: when you access to it at the first time it will fetch the data, not on start. it's possible to make prefetch to start loading on start.